packages/admin/addon/controllers/admin/query-stats.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Controller from '@ember/controller';
+
+export default class AdminQueryStats extends Controller.extend({
+  // anything which *must* be merged to prototype here
+}) {
+  // normal class body definition here
+}
+
+// DO NOT DELETE: this is how TypeScript knows how to look up your controllers.
+declare module '@ember/controller' {
+  interface Registry {
+    'admin/query-stats': AdminQueryStats;
```

Comment:
Not 100% sure, but I think this should be a `.`
```suggestion
    'admin.query-stats': AdminQueryStats;
```

Reply:
hmm I see in users I just have `users`